packages/api/resolvers/mutations/accounts/loginAsGuest.js

```diff
@@ -1,10 +1,16 @@
 import { log } from 'meteor/unchained:core-logger';
-import callMethod from '../../../callMethod';
+import { accountsServer } from 'meteor/unchained:core-accountsjs';
 
-export default function loginAsGuest(root, methodArguments, context) {
+export default async function loginAsGuest(root, methodArguments, context) {
   log('mutation loginAsGuest');
-  return callMethod(context, 'login', {
-    ...methodArguments,
-    createGuest: true,
-  });
+  const {
+    user: { userId },
+    token,
+  } = await accountsServer.loginWithService('guest', methodArguments, context);
```

Comment:
Inconsistently here the context is sent to the accountsserver method but it was not for changePassword and createUser

Reply:
[LoginWithService](https://www.accountsjs.com/docs/api/client/interfaces/transportinterface#loginwithservice) utilizes a different api from [createUser](https://www.accountsjs.com/docs/api/password/classes/accountspassword/#createuser); that's why one of them gets the context as an additional parameter and no the other. As for the hooks, it doesn't really affect us so far, maybe if you'd like the context data to be passed that's a different story.